README.md

```diff
@@ -100,7 +108,38 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+class Animal
+  attr_accessor :name
+  def initialize(name)
+    @name = name
+  end
+  def greet
+    puts "Hi I'm "+ @name
+  end
+  def get_name
+    puts @name
+  end
+  def set_name(new_name)
+    @name = new_name
+  end
```

Comment:
:+1: You took care of the "getting" and "setting" with `attr_accessor`, but these methods can't hurt!